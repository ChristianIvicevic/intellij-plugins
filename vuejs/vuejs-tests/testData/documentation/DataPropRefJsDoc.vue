<template>
  {{ it<caret>ems }}
</template>

<script>
export default {
  data() {
    return {
      /** @type {{name: string, pass: number}[]} */
      items: []
    }
  }
}
</script>