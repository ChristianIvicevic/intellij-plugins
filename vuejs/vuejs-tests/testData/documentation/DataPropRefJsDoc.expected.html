<div class='definition'><pre>items: <span style="">{<br></span><span style="">&#32;&#32;&#32;&#32;name:&#32;</span><span style="color:#000080;">string</span><span style=""><br></span><span style="">&#32;&#32;&#32;&#32;pass:&#32;</span><span style="color:#000080;">number<br></span><span style="">}[]</span></pre></div><table class='sections'><tr><td valign='top'><icon src='AllIcons.FileTypes.JavaScript'/>&nbsp;DataPropRefJsDoc.vue</td></table>